README.md

```diff
@@ -535,6 +535,8 @@ new migration sequence. The arguments it expects are the `direction`
 (e.g. `:up` or `:down`), the migration `kind` (e.g. `:sql`, `:lisp`,
 etc.), an instance of a `provider`, `id`, `description` and `content`.
 
+Knowing `migratum:provider-create-migration` by the documentation below, you can also use the wrapper `migratum.provider.local-path:touch-migration` to create a pair of up-and-down empty migrations with a `LOCAL-PATH` provider.
```

Comment:
Could you please wrap this text into multiple lines, similarly to the existing text?

Reply:
Sure

Reply:
Fixed in [76c8b49](https://github.com/dnaeon/cl-migratum/pull/16/commits/76c8b490d25a7f0da68e6260911610ff12e5b0e7)